lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -294,6 +311,39 @@ IPV_SUCCESS_PAGE:
     pageId: page-ipv-success
   parent: END_JOURNEY
 
+PYI_NEW_DETAILS_PAGE:
+  response:
+    type: page
+    pageId: pyi-new-details
+  events:
+    next:
+      targetState:
+        PYI_CONFIRM_DELETE_DETAILS_PAGE
+    end:
+      targetState:
+        IPV_IDENTITY_REUSE_PAGE
+
+PYI_CONFIRM_DELETE_DETAILS_PAGE:
+  response:
+    type: page
+    pageId: pyi-confirm-delete-details
+  events:
+    reset:
+      targetState:
+        RESET_IDENTITY
```

Comment:
Would like to have an info log in this lambda so we can track user-initiated identity resets (probably also an audit event but TBC). To do so I think we'll need to use `lambdaInput` to distinguish this from the other routes that go to reset-identity. (We'll also need this for the email sending as we only want to send an email for this route)

Could you have a look at how we can do this?

Reply:
Yep, I'll take a look

Reply:
Sounds good, I think we should just do it as part of this ticket